packages/server/modules/workspaces/graph/dataloaders/workspaces.ts

```diff
@@ -0,0 +1,36 @@
+import { db } from '@/db/knex'
+import { getFeatureFlags } from '@/modules/shared/helpers/envHelper'
+import { defineRequestDataloaders } from '@/modules/shared/helpers/graphqlHelper'
+import { getWorkspacesFactory } from '@/modules/workspaces/repositories/workspaces'
+import { WorkspaceWithOptionalRole } from '@/modules/workspacesCore/domain/types'
+import { keyBy } from 'lodash'
+
+const { FF_WORKSPACES_MODULE_ENABLED } = getFeatureFlags()
+
+declare module '@/modules/core/loaders' {
+  interface AllRequestDataLoaders
+    extends Partial<ReturnType<typeof dataLoadersDefinition>> {}
```

Comment:
Making workspaces dataloaders optional, cause they are not gonna be there if the module isn't enabled